Marlin/src/pins/pins_BIGTREE_SKR_MINI_V1_1.h

```diff
@@ -112,13 +111,46 @@
   #define BTN_EN2          PB8
 
   #define LCD_PINS_ENABLE  PB6
-  #define LCD_PINS_D4      PC13
 
-  #if ENABLED(ULTIPANEL)
-    #define LCD_PINS_D5    PB7
-    #define LCD_PINS_D6    PC15
-    #define LCD_PINS_D7    PC14
-  #endif
+  #if ENABLED(FYSETC_MINI_12864)
+
+    #define LCD_BACKLIGHT_PIN -1
+    #define LCD_RESET_PIN  PC13
+    #define DOGLCD_A0      PC12
+    #define DOGLCD_CS      PB6
+    #define DOGLCD_SCK     PB3
+    #define DOGLCD_MOSI    PB5
+
+    #define FORCE_SOFT_SPI   // SPI MODE3
+
+    #define LED_PIN        PB7   // red pwm
+    //#define LED_PIN        PC15   // green
+    //#define LED_PIN        PC14   // blue
+
+    //#if EITHER(FYSETC_MINI_12864_1_2, FYSETC_MINI_12864_2_0)
+    //  #ifndef RGB_LED_R_PIN
+    //    #define RGB_LED_R_PIN PB7
+    //  #endif
+    //  #ifndef RGB_LED_G_PIN
+    //    #define RGB_LED_G_PIN PC15
+    //  #endif
+    //  #ifndef RGB_LED_B_PIN
+    //    #define RGB_LED_B_PIN PC14
+    //  #endif
+    //#elif ENABLED(FYSETC_MINI_12864_2_1)
+    //  #define NEOPIXEL_PIN    PB7
+    //#endif
```

Comment:
@tpruvot — Do you think these would work, or is just the one LED supported?

Reply:
PIN: 23 PB7         LED_PIN                                 TIM4 CH2
PIN: 46 PC14         <unused/unknown>                       Input  = 0
PIN: 47 PC15         <unused/unknown>                       Input  = 0

PC14/15 dont have a timer to handle the pwm required for the RGB...

Reply:
We allow for non-PWM RGB which can set one of 7 colors at a single brightness.